versions.yaml

```diff
@@ -158,9 +158,9 @@ assets:
     version: "v5.4.15"
 
   kernel-experimental:
-    description: "Linux kernel with virtiofs 3.0"
+    description: "Linux kernel with virtiofs-dev"
     url: "https://gitlab.com/virtio-fs/linux.git"
-    tag: "virtio-fs-v0.3"
+    tag: "virtio-fs-dev"
```

Comment:
btw, could we use a commit ID here, instead of the branch name? So that we can get reproducible builds over time.

Reply:
@chavafg Good suggestion. I have included a commit here instead. ptal.

Reply:
@chavafg Looks like `install_kernel` script in packaging repo expects a tag for the experimental kernel. I have reverted back to using a branch. Lets use a commit id in a future PR after we fix the script. Wdyt?

Reply:
ok, sure